### ISSUE TITLE
Support AllNamespace in get <resource> commands with the k8s module

### ISSUE THREAD

**Issue by djsly** [gruntwork-io/terratest#478]
Or maybe add support for ListNamespaces, such that we could itterates over them

We wanted to write a test that would get all pods in the cluster and ensure that they are in Running State. 

this isn't currently possible unless I hardcode a list of namespace to look at.

**Comment by yorinasub17** [comment]
This isn't well documented, but you can use `k8s.ListPods` with `KubectlOptions.Namespace` set to `""` to get all pods.

**Comment by adhaamehab** [comment]
Hi, I'll be working on this :)

**Comment by djsly** [comment]
correct, `""` works well indeed, I guess it is just missing proper documentation :)

**Comment by james03160927** [comment]
Duplicate of https://github.com/gruntwork-io/terratest/issues/1540.
